ci.py

```diff
@@ -178,12 +206,158 @@ def kill(signal, frame):
         sys.exit(p.returncode)
 
 
+def update_docker_compose(modules):
+    try:
+        with open("docker-compose.yml") as compose_file:
+            compose_dict = yaml.load(compose_file)
+    except:
+        raise FileReadException('Error reading chart yaml for changed chart')
+    if modules:
+        compose_services = compose_dict['services']
+        for module in modules:
+            service_name = MODULE_TO_COMPOSE_SERVICE[module]
+            services_to_update = []
```

Comment:
I believe these 5 lines could be written more simply as:
services_to_update = service_name.split(',')